src/Humanizer/Localisation/TimeUnit.cs

```diff
@@ -13,7 +13,8 @@ public enum TimeUnit
         Day,
         Week,
         Month,
-        Year
+        Year,
+        Never
```

Comment:
I agree, it is a problem passing `Never` into `TimeSpan` humanization, which I didn't notice while working on the `DateTime` humanization.